common/script/index.js

```diff
@@ -2063,23 +2063,33 @@ api.wrap = function(user, main) {
         return content.gear.flat[type + "_base_0"];
       }
       return item;
-    },
+    },   
     handleTwoHanded: function(item, type, req) {
-      var message, ref, weapon;
+      var message, ref, weapon, shield;
       if (type == null) {
         type = 'equipped';
       }
       if (item.type === "shield" && ((ref = (weapon = content.gear.flat[user.items.gear[type].weapon])) != null ? ref.twoHanded : void 0)) {
         user.items.gear[type].weapon = 'weapon_base_0';
-        message = i18n.t('messageTwoHandled', {
+        message = [i18n.t('messageTwoHandled', {
```

Comment:
Why not just change this message to include the information about un-equipping? That would eliminate the need to change this to an array.